File: VB/WorkflowDemoEF.Module/Objects/Task.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Base
Imports DevExpress.ExpressApp
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations

Namespace WorkflowDemoEF.Module.Objects
    <DefaultClassOptions, ImageName("BO_Task")> _
    Public Class Task
        Private privateId As Integer
        <Browsable(False)> _
        Public Property Id() As Integer
            Get
                Return privateId
            End Get
            Private Set(ByVal value As Integer)
                privateId = value
            End Set
        End Property
        Public Property Subject() As String
        Public Overridable Property Issue() As Issue
    End Class
End Namespace

File: CS/WorkflowDemoEF.Module/Objects/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Base;
using DevExpress.ExpressApp;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace WorkflowDemoEF.Module.Objects {
	[DefaultClassOptions]
    [ImageName("BO_Task")]
    public class Task {
        [Browsable(false)]
        public int Id { get; private set; }
        public string Subject { get; set; }
        public virtual Issue Issue { get; set; }
	}
}
